pkg/txn/storage/txn/catalog.go

```diff
@@ -61,7 +71,7 @@ func (d DatabaseRow) AttrByName(tx *Transaction, name string) (ret Nullable, err
 			if attr.Name != name {
 				continue
 			}
-			if !typeMatch(ret.Value, attr.Type.Oid) {
+			if !memtable.TypeMatch(ret.Value, attr.Type.Oid) {
 				panic(fmt.Errorf("%s should be %v typed", name, attr.Type))
```

Comment:
Clean up old code when you see one. :)

Reply:
These are for panic, I will use Sprintf instead.